addons/web/static/src/css/base.sass

```diff
@@ -2124,9 +2124,10 @@ $sheet-padding: 16px
             padding: 0 !important
             cursor: ns-resize
             .oe_list_handle
-                font-size: 18px
+                font-size: 7px
                 color: #E0E0E0
-                margin-right: 7px
+                margin-right: 10px
+                margin-top: 8px
```

Comment:
could use vertical-align on the `oe_list_field_handle` class instead.
You can't use text-center since oe_number force the right align with `!important`, so check with sle if he is ok with your horizontal centering ;)
:boar: !